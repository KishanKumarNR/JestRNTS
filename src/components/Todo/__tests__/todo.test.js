import React, {Component} from 'react';
import renderer from 'react-test-renderer';
import HelloWorldTodo from "../index";


test('renders correctly', () => {
    const tree = renderer.create(<HelloWorldTodo />).toJSON();
    expect(tree).toMatchSnapshot();
});

test("something", () => {
    expect("this").not.toEqual("that");
})

it('renders correctly', () => {
    const tree = renderer
        .create(<HelloWorldTodo/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});