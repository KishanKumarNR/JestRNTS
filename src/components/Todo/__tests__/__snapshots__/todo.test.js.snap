// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
    Hello world
  </Text>
</View>
`;

exports[`renders correctly 2`] = `
<View>
  <Text>
    Hello world
  </Text>
</View>
`;
